test/spirv-unknown-extensions.spt

```diff
@@ -0,0 +1,44 @@
+119734787 65536 393230 16 0 
```

Comment:
There's stray whitespace at the ends of many of the lines in this test. If you push a new version, it would be good to fix this.

Reply:
I believe those whitespaces were inserted by `llvm-spirv` tool.

Reply:
Fixed